kubernetes/resource_kubernetes_persistent_volume_v1.go

```diff
@@ -240,14 +240,22 @@ func resourceKubernetesPersistentVolumeV1Create(ctx context.Context, d *schema.R
 			}
 
 			statusPhase := fmt.Sprintf("%v", out.Status.Phase)
-			log.Printf("[DEBUG] Persistent volume %s status received: %#v", out.Name, statusPhase)
+			statusMessage := fmt.Sprintf("%v", out.Status.Message)
+			if statusMessage == "" {
+				log.Printf("[DEBUG] Persistent volume %s status received: %#v", out.Name, statusPhase)
+			} else {
```

Comment:
if message is not None add it to log.
` out.Status.Reason` is a CamelCase string so does not make sense to display in log